randovania/games/prime1/json_data/Tallon Overworld.txt

```diff
@@ -551,9 +551,13 @@ Extra - aabb: [-703.2118, 297.52924, -49.04736, -619.3062, 378.77765, 64.30938]
       Trivial
   > Pickup (Missile Expansion)
       All of the following:
-          # https://youtu.be/x2zjlvSy0Og
           Space Jump Boots
-          Combat/Scan Dash (Intermediate) or Use Grapple Beam
+          Any of the following:
+              # https://youtu.be/x2zjlvSy0Og
```

Comment:
If the trick is on it's own without an `and` or `or` box to place a comment, please add one so that the comment only applies to the trick.

This reads as the video applying to both the vanilla method and the dash, which can mess with the video directory script in certain edge cases.

Reply:
Understood, separated in newest commit.